### ISSUE TITLE
[GO] Cannon - Bug with Faldir's Cove, Blackwater pirate ship

### ISSUE THREAD

**Issue by shamblr** [dalaranwow/dalaran-wow#3509]
**Description**: I arrived in Faldir's cove with awe as I had never been to that area in retail. My first thought was to board the boat parked there. I made it into it's interior fine, but as I exited I was killed randomly. I thought perhaps an Npc had killed me. However doing the quest for Shakes O'Breen (http://www.wowhead.com/npc=2610/shakes-obreen) Death From Below (http://www.wowhead.com/quest=26628/death-from-below) I found the bug to happen again. This time I boarded the ship seeing a cannon I figured this could be interacted with. As I approached it however I once again was killed, but managed to complete the quest regardless. 

**Current behaviour**: In Faldir's Cove boarding the blackwater pirate ship and approaching the cannon in the corner instantly kills your character. This causes no durability loss however. This can be repeated numerous times as pleased and happens on command. 

**Expected behaviour**: I expect nothing should happen, especially not death.

**Steps to reproduce the problem**:
1. Arrive in Faldir's cove, board pirate ship.
2. Face cannon from behind.
3. Approach cannon and die. 

**Proof**
https://gyazo.com/e2e6588ad44feb850e77d7cb0f7ebab7

https://gyazo.com/d9a7ccef8ea3a1efd7e891127b3000cd (After my first attempt, no damage on my gear after the death)

https://gyazo.com/11e3b6d2ef652fe8ba2d7668087f5df2 (I'm using a bomb to display the area that you die in, you can see my skeleton from the first attempt. 

I caught these on desktop capture, however I felt sending you movie files to preview would be too cumbersome. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Stihii** [comment]
The issue happened to me too.

**Comment by Golganeth** [comment]
#731 - 
> - **Issue 2:** Upon using the lone cannon on the main deck, *World Invisible Trigger's Cannon Ball* instantly kills you.
- **Note:** Even when you don't use the cannon within the boat, you may get instantly killed by the trigger regardless.

**Comment by PSAlias** [comment]
I have also experienced this bug while poking around the area. Thing is, I'm fairly certain this bug was in retail and I'm not sure if they had fixed it by 3.3.5

**Comment by Rushor** [comment]
DB/Quest:  Death From Below - Cannon no longer instantkills the player

* closes https://github.com/dalaranwow/dalaran-wow/issues/3509
* commit: 38e96fa13b5ccc3af03fc83bcecaa922b0aacb17
* commit: fdb6222e64aae5b6b5ea12ed50604d8c8ca0fdda


The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
